test/travis.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+set -u
+set -x
+
+# These are steps to run on TravisCI under a containerized Ubuntu system
+make
```

Comment:
should we detect the number of cpus the system has using getconf and add that to -j here ?